easybuild/easyconfigs/r/R-tesseract/R-tesseract-5.1.0-foss-2022a-R-4.2.1.eb

```diff
@@ -0,0 +1,58 @@
+easyblock = 'Bundle'
+
+name = 'R-tesseract'
+version = '5.1.0'
+versionsuffix = '-R-%(rver)s'
```

Comment:
@smoors do we still need the `versionsuffix`?

Reply:
good question :)
a quick search gives me 6 easyconfigs without and 22 with the versionsuffix..

Reply:
I'm happy to go ahead with this as it is, because:
* we've got easyconfigs with and without the versionsuffix
* we neither enforce that easyconfigs should be with or without versionsuffix

If we do want to do so then we can standardise them then.